styles/main.css

```diff
@@ -73,8 +73,14 @@ main,
   overflow-y: auto;
   margin-bottom: 10px;
   height: calc(100% - 80px);
-  max-height: 900px;
 }
+
+@media only screen and (min-width: 769px) {
```

Comment:
Yea, just read up on that. I guess it was to prevent older browsers from seeing media queries.